src/envoy/http/service_control/filter_stats.h

```diff
@@ -27,47 +27,71 @@ namespace service_control {
  * All stats for the service control filter. @see stats_macros.h
  */
 
-// TODO(taoxuy): add macro function to init [check|report]_count_STATUS
 // clang-format off
-#define ALL_SERVICE_CONTROL_FILTER_STATS(COUNTER, HISTOGRAM) \
+
+#define SERVICE_CONTROL_FILTER_STATS(COUNTER, HISTOGRAM) \
   COUNTER(allowed)                                           \
   COUNTER(denied)                                            \
   HISTOGRAM(request_time, Milliseconds)                      \
   HISTOGRAM(backend_time, Milliseconds)                      \
-  HISTOGRAM(overhead_time, Milliseconds)                     \
-  COUNTER(check_count_OK)                                    \
-  COUNTER(check_count_CANCELLED)                             \
-  COUNTER(check_count_UNKNOWN)                               \
-  COUNTER(check_count_INVALID_ARGUMENT)                      \
-  COUNTER(check_count_DEADLINE_EXCEEDED)                     \
-  COUNTER(check_count_NOT_FOUND)                             \
-  COUNTER(check_count_ALREADY_EXISTS)                        \
-  COUNTER(check_count_PERMISSION_DENIED)                     \
-  COUNTER(check_count_RESOURCE_EXHAUSTED)                    \
-  COUNTER(check_count_FAILED_PRECONDITION)                   \
-  COUNTER(check_count_ABORTED)                               \
-  COUNTER(check_count_OUT_OF_RANGE)                          \
-  COUNTER(check_count_UNIMPLEMENTED)                         \
-  COUNTER(check_count_INTERNAL)                              \
-  COUNTER(check_count_UNAVAILABLE)                           \
-  COUNTER(check_count_DATA_LOSS)                             \
-  COUNTER(check_count_UNAUTHENTICATED)                       \
-  COUNTER(report_count_OK)                                   \
-  COUNTER(report_count_CANCELLED)                            \
-  COUNTER(report_count_UNKNOWN)                              \
-  COUNTER(report_count_INVALID_ARGUMENT)                     \
-  COUNTER(report_count_DEADLINE_EXCEEDED)                    \
-  COUNTER(report_count_NOT_FOUND)                            \
-  COUNTER(report_count_ALREADY_EXISTS)                       \
-  COUNTER(report_count_PERMISSION_DENIED)                    \
-  COUNTER(report_count_RESOURCE_EXHAUSTED)                   \
-  COUNTER(report_count_FAILED_PRECONDITION)                  \
-  COUNTER(report_count_ABORTED)                              \
-  COUNTER(report_count_OUT_OF_RANGE)                         \
-  COUNTER(report_count_UNIMPLEMENTED)                        \
-  COUNTER(report_count_INTERNAL)                             \
-  COUNTER(report_count_UNAVAILABLE)                          \
-  COUNTER(report_count_DATA_LOSS)                            \
+  HISTOGRAM(overhead_time, Milliseconds)
+
+// TODO(taoxuy): add macro function to init [check|quota|report]_count_STATUS
+#define SERVICE_CONTROL_CHECK_STATS(COUNTER, HISTOGRAM) \
```

Comment:
can we remove HISTOGRAM in this macro since it is not used?

Reply:
done